test/api.js

```diff
@@ -705,9 +705,16 @@ function generateTests(prefix, apiCreator) {
 	test(prefix + 'caching is enabled by default', function (t) {
 		t.plan(3);
 		rimraf.sync(path.join(__dirname, 'fixture/caching/node_modules'));
+
+		var prevCwd = process.cwd();
+		process.chdir(path.join(__dirname, 'fixture/caching'));
```

Comment:
The `pkgDir` and `resolveTestsFrom` directories can be passed to `Api` though. That's how its meant to be used, [implicit defaults](https://github.com/avajs/ava/issues/611) aside. We shouldn't need to change the working directory of the `tap` process.